src/sections/user/user-table-row.tsx

```diff
@@ -13,25 +13,27 @@ import MenuItem, { menuItemClasses } from '@mui/material/MenuItem';
 import { Label } from 'src/components/label';
 import { Iconify } from 'src/components/iconify';
 
-// ----------------------------------------------------------------------
+// ----------------------------------------------------------------
 
 export type UserProps = {
   id: string;
   name: string;
   role: string;
   status: string;
   company: string;
-  avatarUrl: string;
+  avatarUrl?: string; // Optional avatarUrl property
   isVerified: boolean;
 };
 
 type UserTableRowProps = {
```

Comment:
this is also repeated declaration
you can keep those attributes optional which are not used everywhere
thus declare only a common interface

Reply:
still it is a repeated interface declaration
Declare this ```UserTableRowProps``` interface in ```src/inteface``` folder, and you can keep those fields optional which are not use everywhere.